chainer/iterators/multiprocess_iterator.py

```diff
@@ -425,6 +459,12 @@ def _allocate_shared_memory(self):
                 sharedctypes.RawArray('b', self.batch_size * self.mem_size)
 
     def launch_thread(self):
+        # Defer memory allocation due to pickling issues.
```

Comment:
It looks this comment is about "the change", not "the code".
This call itself does not defer the memory allocation.
How about something like "Memory allocation is deferred until the launch of the threads due to ..."?

Reply:
With the new changes, deferring the memory allocation was not needed, so I reverted it :)